apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.sql

```diff
@@ -50,9 +50,10 @@ FROM
   LEFT JOIN assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
   LEFT JOIN assessments AS a ON (a.id = ai.assessment_id)
   LEFT JOIN pl_courses AS c ON (c.id = v.course_id)
```

Comment:
We should always expect to have a corresponding course, so you might consider making this a `JOIN` instead of a `LEFT JOIN` and moving it up below `JOIN questions ...`.